templates/launch.html

```diff
@@ -164,12 +164,12 @@ <h3>Runner Data</h3>
 
             <div class="field-container">
                 <label for="account_service_log_out_url">Account Service Log Out URL</label>
-                <input id="account_service_log_out_url" name="account_service_log_out_url" type="text" value="{{.AccountServiceLogOutURL}}" class="qa-account_service_log_out_url">
+                <input id="account_service_log_out_url" type="text" value="{{.AccountServiceLogOutURL}}" class="qa-account_service_log_out_url">
```

Comment:
How come we don't send this one?

Reply:
We didn't send it because it's not included in the ons-schema-definitions doc, so I removed it because I didn't think it needed to be in the token

Reply:
In fact, it's included as an optional field, but that's only in the v1 payload document not in the v2 payload doc, so since we are planning to remove v1, I guess this would be appropriate?

Reply:
Hmm it should be removed when the ticket to remove v1 gets picked up, but seems like the type of thing that could be easily missed